lib/restful_resource/base.rb

```diff
@@ -54,7 +56,7 @@ def self.delete(id, **params)
 
     def self.put(id, data: {}, headers: {}, **params)
       params_without_options, options = format_params(params)
-      options.delete(:headers)
+      headers.merge! options.delete(:headers)
```

Comment:
@isaacseymour could you refresh our memory why we were deleting the headers in the first place?

Reply:
Mmmmm yeah. I can't totally remember. I'm not sure we worked it out, I think we were just like...seems weird, let's change it. 😂